RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -1400,8 +1400,8 @@ All our stargazers, likers &amp; followers, for the warm fuzzies
   <data name="ParserState_ResolverError" xml:space="preserve">
     <value>Resolver Error</value>
   </data>
-  <data name="ParserState_Resolving" xml:space="preserve">
```

Comment:
Hmm, right.  I don't know German or French...  @Vogel612 Can you check this too?